internal/utils/rand.go

```diff
@@ -1,22 +0,0 @@
-package utils
-
-import (
-	"math/rand"
-)
-
-// RandomAlphaNum generates random alphanumeric string
-// in case length <= 0 it returns empty string
-func RandomAlphaNum(length int) string {
```

Comment:
I think it's used in `timelock_test.go`

Reply:
there are no failures on CI because it has special tag +integration which is not touched when running tests (yet)